drupal/profiles/favrskov/modules/custom/favrskov_helper/favrskov_helper.module

```diff
@@ -5,6 +5,8 @@
  * Main file of favrskov_helper.module
  */
 
+require_once dirname(__FILE__) . '/favrskov_helper.field.inc';
```

Comment:
Actually *.file.inc are not supported properly, comment above can be disregarded.